ConfCore/Adapter.swift

```diff
@@ -7,46 +7,28 @@
 //
 
 import Foundation
-import SwiftyJSON
 
-enum AdapterError: Error {
-    case invalidData
-    case unsupported
-    case missingKey(JSONSubscriptType)
+// This is needed to keep the same behavior that adapters had before
+// Where it could adapt array of items even if some of the individual items failed to adapt
```

Comment:
@otaviolima I understand that you're keeping original code behavior, but I'm wondering whether that behavior is truly needed?